Src/WitsmlExplorer.Api/Jobs/BatchModifyWellJob.cs

```diff
@@ -0,0 +1,9 @@
+using WitsmlExplorer.Api.Models;
+
+namespace WitsmlExplorer.Api.Jobs
+{
+    public class BatchModifyWellJob
+    {
+        public Well[] Wells { get; set; }
```

Comment:
I see, But along with reference-id's we need to pass other parameters like (name, country, timezone, field etc...) for multiple wells. So it would be array of Well DTO's

Reply:
I think it's correct to use Well here as we are passing multiple parameters, like Chandra said. The same approach is used in for instance `ModifyWellJob.cs` and `CreateWellJob.cs`.